api/controllers/main.js

```diff
@@ -1,14 +1,22 @@
 const fs = require('fs');
 const path = require('path');
+const SiteWideErrorLoader = require('../services/SiteWideErrorLoader');
 
 const manifestPath = path.join(__dirname, '..', '..', 'webpack-manifest.json');
 const webpackAssets = JSON.parse(fs.readFileSync(manifestPath, 'utf-8'));
 
 module.exports = {
   index(req, res) {
-    res.render('index.html', {
+    const context = {
+      siteWideError: null,
```

Comment:
I ended up needing this here or I'd get a `ReferenceError` from EJS when there wasn't an error to display.